test/src/tests-identities-attributes.ts

```diff
@@ -951,41 +1034,41 @@ describe('identities and attributes', function() {
                 email: 'initial@gmail.com',
             },
         };
-
+        mParticle.config.flags.eventBatchingIntervalMillis = 5000
         mParticle.init(apiKey, window.mParticle.config);
-
+        waitForCondition(hasIdentifyReturned)
+        .then(() =>  {
+        mParticle.upload();
         expect(
             JSON.parse(`${fetchMock.lastOptions().body}`).user_identities
         ).to.have.property('email', 'initial@gmail.com');
-
+        
         mParticle.logEvent('testAfterInit');
-
+        mParticle.upload();
+        
         expect(
             JSON.parse(`${fetchMock.lastOptions().body}`).user_identities
         ).to.have.property('email', 'initial@gmail.com');
 
-        fetchMock.calls().forEach(call => {
```

Comment:
Why is this loop being removed?

Reply:
i meant to add a comment to this PR.  basically i forced this to all be under a single batch instead of each batch uploading individually, which isn't a realistic use case.  since it's not a single batch, we no longer need to loop through every call